Windows.md

```diff
@@ -0,0 +1,82 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# Building parquet-cpp on Windows
+
+## Fast setup of building requirements with conda and conda-forge
+
+To not waste a lot of time to setup the thirdparty build dependecies for
+parquet-cpp, tested and recommended way is to use conda package manager.
+Please feel free to extend this document with others ways to setup
+development environment for parquet-cpp.
+
+### conda and package toolchain
+
+[Miniconda][1] is a minimal Python distribution including the conda package
+manager. To get started, download and install a 64-bit distribution.
+
+We recommend using packages from [conda-forge][2].
+Launch cmd.exe and execute following commands:
+
+```shell
+conda config --add channels conda-forge
+```
+
+Now, you can bootstrap a build environment
+
+```shell
+conda create -n parquet-dev cmake git boost curl zlib
```

Comment:
Is Snappy for Windows available in defaults (repo.continuum.io)? Support I could look myself =)